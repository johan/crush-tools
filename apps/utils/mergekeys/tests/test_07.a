Month	Site	Ad Size	Ad Type	Impressions	Clicks
June	Page 3	468x60	Standard	1008405	9
June	Page 3	Not Selected	Standard	160376	527
June	SalesManager_test	728x90	Exclusive	3229	30
June	TheLondonSite	125x125	Exclusive	1042	25
