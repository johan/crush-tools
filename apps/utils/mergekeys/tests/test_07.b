Month	Site	Ad Size	Ad Type	Activity-Type
June	Page 3	468x60	Standard	abc123
June	Page 3	468x60	Standard	xzy987
June	Page 3	Not Selected	Time Based Bulk	abc123
June	Page 3	Not Selected	Time Based Bulk	xyz987
June	Page 3	Not Selected	Time Based Bulk	deadbeef
June	TheLondonSite	125x125	Exclusive	abc123
June	TheLondonSite	125x125	Exclusive	deadbeef
