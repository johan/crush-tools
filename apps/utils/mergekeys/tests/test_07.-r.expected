Month	Site	Ad Size	Ad Type	Impressions	Clicks	Activity-Type
June	Page 3	468x60	Standard	1008405	9	abc123
June	Page 3	468x60	Standard	1008405	9	xzy987
June	Page 3	Not Selected	Time Based Bulk			abc123
June	Page 3	Not Selected	Time Based Bulk			xyz987
June	Page 3	Not Selected	Time Based Bulk			deadbeef
June	TheLondonSite	125x125	Exclusive	1042	25	abc123
June	TheLondonSite	125x125	Exclusive	1042	25	deadbeef
